File: templates/web/backend/rust/axum/sources/status.rs
use serde::Serialize;

#[derive(Serialize)]
pub struct Status {
    pub service: String,
    pub version: String,
}

pub fn get_status() -> Status {
    Status {
        service: "{{project_name}}".to_string(),
        version: "0.1.0".to_string(),
    }
}

File: templates/web/backend/python/django/sources/status.py
import platform


class Status:
    def __init__(self):
        self.service = "{{project_name}}"
        self.version = "0.1.0"
        self.python_version = platform.python_version()

    def dict(self):
        return {
            "service": self.service,
            "version": self.version,
            "python_version": self.python_version,
        }
